src/plugins/email/test_email.h

```diff
@@ -0,0 +1,71 @@
+/**
+ * @file
+ *
+ * @brief Tests for email plugin
+ *
+ * @copyright BSD License (see doc/LICENSE.md or https://www.libelektra.org)
+ *
+ */
+
+#include <stdio.h>
+
+#include <tests_plugin.h>
+
+#include <kdbconfig.h>
+
+#ifdef HAVE_FEATURES_H
+
+#include <features.h>
+
+#endif // HAVE_FEATURES_H
+
+static void testEmail (char const * const email, const int ret)
+{
+	Key * parentKey = keyNew ("user:/tests/email", KEY_VALUE, "", KEY_END);
+	KeySet * conf = ksNew (0, KS_END);
+	KeySet * ks = ksNew (10, keyNew ("user:/tests/email/totest", KEY_VALUE, email, KEY_META, "check/email", KEY_END), KS_END);
```

Comment:
```suggestion
	KeySet * ks = ksNew (10, keyNew ("user:/tests/email/totest", KEY_VALUE, email, KEY_META, "check/email", "", KEY_END), KS_END);
```

You are missing the second argument for `KEY_META`. This causes an abort in `keyVNew` (see below) because `KEY_END` is interpreted as the second `char *` and then it reads beyond the argument list and tells you that whatever random memory it read is not a correct option.

https://github.com/ElektraInitiative/libelektra/blob/4da0d2a9e5ebb5e167265f7141d7a0ac89ed46db/src/libs/elektra/key.c#L232

This can be hard to reproduce because if the memory after `KEY_END` is just zero there is no abort.